documentation/documentation-pointers.md

```diff
@@ -0,0 +1,47 @@
+## Overview about documentation for tagging documents
+
+In  general one can use the command `texdoc <package-name>` on the command line to get documentation about a specific package. Instead of using `texdoc` on your machine you can also use the online version at [https://texdoc.org](https://texdoc.org/index.html).
+
+
+The following list gives an overview about different packages related to tagging and some details on the contents of each:
+
++ `ltnews.pdf` (or `ltnews<number>.pdf`)
```

Comment:
I'd probably make all these texdoc.org links, so people can just clock through rather than having to copy these filenames

Reply:
good idea